opentreemap/treemap/templatetags/form_extras.py

```diff
@@ -386,7 +386,7 @@ def _field_value(model, field_name, data_type):
             units = get_units_if_convertible(
                 model.instance, object_name, field_name)
             if units != '':
-                units = get_unit_name(units)
+                units = get_unit_abbreviation(units)
```

Comment:
Using full unit names got introduced by accident to fix the problem of `sq_m` being displayed. If we want to consider showing full unit names instead of abbreviations we should open a larger issue, but we use abbreviations everywhere and that's been fine. This is really just a less-drastic fix for the problem of `sq_m` being displayed.